Traceback (most recent call last):
  File "/home/runner/.cache/pypoetry/virtualenvs/mtbp3-STzYEG9e-py3.10/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/runner/.cache/pypoetry/virtualenvs/mtbp3-STzYEG9e-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/runner/.cache/pypoetry/virtualenvs/mtbp3-STzYEG9e-py3.10/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/runner/.cache/pypoetry/virtualenvs/mtbp3-STzYEG9e-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/runner/.cache/pypoetry/virtualenvs/mtbp3-STzYEG9e-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/runner/.cache/pypoetry/virtualenvs/mtbp3-STzYEG9e-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
print("\n".join(ctoc.find_section_given_words(words=["REMS","dsur"], outfmt='tree')))
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[8], line 1[0m
[0;32m----> 1[0m [38;5;28mprint[39m([38;5;124m"[39m[38;5;130;01m\n[39;00m[38;5;124m"[39m[38;5;241m.[39mjoin([43mctoc[49m[38;5;241;43m.[39;49m[43mfind_section_given_words[49m[43m([49m[43mwords[49m[38;5;241;43m=[39;49m[43m[[49m[38;5;124;43m"[39;49m[38;5;124;43mREMS[39;49m[38;5;124;43m"[39;49m[43m,[49m[38;5;124;43m"[39;49m[38;5;124;43mdsur[39;49m[38;5;124;43m"[39;49m[43m][49m[43m,[49m[43m [49m[43moutfmt[49m[38;5;241;43m=[39;49m[38;5;124;43m'[39;49m[38;5;124;43mtree[39;49m[38;5;124;43m'[39;49m[43m)[49m))

File [0;32m~/work/mtbp3/mtbp3/mtbp3/health/ectd.py:65[0m, in [0;36mctoc_by_fda.find_section_given_words[0;34m(self, words, outfmt, include, to_right)[0m
[1;32m     63[0m [38;5;28;01mfor[39;00m row [38;5;129;01min[39;00m out:
[1;32m     64[0m     [38;5;28;01mfor[39;00m word [38;5;129;01min[39;00m words:
[0;32m---> 65[0m         start [38;5;241m=[39m [43mrow[49m[38;5;241;43m.[39;49m[43mlower[49m[43m([49m[43m)[49m[38;5;241;43m.[39;49m[43mindex[49m[43m([49m[43mword[49m[38;5;241;43m.[39;49m[43mlower[49m[43m([49m[43m)[49m[43m)[49m
[1;32m     66[0m         end [38;5;241m=[39m start [38;5;241m+[39m [38;5;28mlen[39m(word)
[1;32m     67[0m         row [38;5;241m=[39m row[:start] [38;5;241m+[39m [38;5;124mf[39m[38;5;124m"[39m[38;5;130;01m\033[39;00m[38;5;124m[91m[39m[38;5;132;01m{[39;00mrow[start:end][38;5;132;01m}[39;00m[38;5;130;01m\033[39;00m[38;5;124m[0m[39m[38;5;124m"[39m [38;5;241m+[39m row[end:]

[0;31mValueError[0m: substring not found

